alonet/deformable_detr/deformable_detr.py

```diff
@@ -167,14 +168,15 @@ def __init__(
         if device is not None:
             self.to(device)
 
-        if weights is not None and (
-            weights in ["deformable-detr-r50", "deformable-detr-r50-refinement"]
-            or ".pth" in weights
-            or ".ckpt" in weights
-        ):
-            alonet.common.load_weights(self, weights, device, strict_load_weights=strict_load_weights)
-        else:
-            raise ValueError(f"Unknown weights: '{weights}'")
+        if weights is not None:
+            if weights is not None and (
```

Comment:
Maybe is not necessary make the question "weights is not None" again

Reply:
yes, done